trucks.js

```diff
@@ -501,4 +502,52 @@ var foodTrucks = [
 		Facebook: 'https://www.facebook.com/fulltilticecreams',
 		Twitter: 'https://twitter.com/fticecream'
 	}
-];
\ No newline at end of file
+];
+
+function getTrucks() {
+	return _.map(foodTrucks, function(item) {
```

Comment:
While this works, it's an unnecessary step. You can simply return foodTrucks directly as it's already an array.